Makefile

```diff
@@ -7,6 +7,8 @@ IMAGE_NAME ?= ceph-csi-operator
 # Allow customization of the name prefix and/or namespace
 NAME_PREFIX ?= ceph-csi-operator-
 NAMESPACE ?= $(NAME_PREFIX)system
+# A comma separated list of namespaces for operator to cache objects from
+WATCH_NAMESPACE ?= ""
```

Comment:
use `NAMESPACE` as default instead of an empty string?

Reply:
I suppose we can rename this to ADDITIONAL_WATCH_NAMESPACES or always have operator ns in this.

I didn't go with latter as the ns is known by downward api which has different semantics and so left this as user configurable option.